assessment_software/src/app/model/assignment-card/assignment-card.ts

```diff
@@ -0,0 +1,27 @@
+export class AssignmentCard {
+  public id: string;
+  public title: string;
+  public submissions: string;
+  public deadline: Date;
+  public testcases: string;
+  public share: string;
+  public edit: string;
+
+  constructor(
+    id: string,
+    title: string,
+    submissions: string,
+    deadline: Date,
+    testcases: string,
+    share: string,
+    edit: string
+  ) {
+    this.id = id;
+    this.title = title;
+    this.submissions = submissions;
+    this.deadline = deadline;
```

Comment:
what if we use a custom type for this property, such as a "DateRange" type? To make it easier to represent deadlines that span multiple days.

Reply:
We can make it like that, but I don't think we will have a use case for it. I would say it is better to keep it simple and make this work then extend it?